pgxn/hnsw/Makefile

```diff
@@ -2,7 +2,7 @@ EXTENSION = hnsw
 EXTVERSION = 0.1.0
 
 MODULE_big = hnsw
-DATA = $(wildcard *--*.sql)
+DATA = hnsw--0.1.0.sql
```

Comment:
Looks like unrelated change, but generally wildcard is used to have a list of all files, i.e. if there are version upgrades like `hnsw--0.1.0--0.2.0.sql, iirc

Reply:
Yeah it is unrelated, I was using it to test roles nad noticed I couldn't actually do `CREATE EXTENSION hnsw` because the `hnsw--0.1.0.sql` was not being installed.

Reply:
Removed this since it's unrelated to this PR